addons/account/models/account_tax.py

```diff
@@ -1193,6 +1193,9 @@ def grouping_key_generator(base_line, tax_values):
             amount_tax += sum(x['tax_group_amount'] for x in groups_by_subtotal[subtotal_title])
 
         amount_total = amount_untaxed + amount_tax
+        rounding_line = base_line['record'].move_id.line_ids.filtered(lambda l: l.display_type == 'rounding')
```

Comment:
You actually can't be sure that `base_line['record']` is a `account.move` as this function is also used in Sales/Purchase/etc.
So maybe add an optional parameter to  `_prepare_tax_totals` and only fill it when calling from `account.move`, or add the new key `formatted_amount_total_rounded` in the result only after calling this function in `_compute_tax_totals`

Reply:
I just pushed at the same time. I'll change the fix based on your remarks

Reply:
Maybe the second option. That's what I wanted first. And knowing what you said, it feels better